gym_ignition/__init__.py

```diff
@@ -43,21 +44,25 @@
 register(
     id='CartPoleGymppy-Discrete-v0',
     entry_point='gym_ignition.base.gazebo_env:GazeboEnv',
+    max_episode_steps=5000,
     kwargs={'task': cartpole_discrete.CartPoleDiscrete,
             'robot': sim.cartpole.CartPoleRobot,
             'sdf': "CartPole/CartPole.sdf",
             'world': "DefaultEmptyWorld.world",
+            'rtf': 1E9,
```

Comment:
Not blocking, but if we could have some kind of macro/option that indicates "do not wait" instead of the 1e9 probably it is more readable.

Reply:
Not a big fan of macro, I'll be using the max `float` number.